rktl_perception/launch/camera.launch

```diff
@@ -50,12 +50,13 @@
         <!-- Ball tracking -->
         <node pkg="rktl_perception" type="ball_detection" name="ball_detection">
             <param name="publishThresh" value="true" />
-            <param name="min_hue" value="55" />
-            <param name="min_sat" value="30" />
-            <param name="min_vib" value="30" />
+            <param name="min_hue" value="50" />
             <param name="max_hue" value="100" />
-            <param name="max_sat" value="200" />
-            <param name="max_vib" value="80" />
+            <param name="min_sat" value="70" />
+            <param name="max_sat" value="180" />
+            <param name="min_vib" value="40" />
+            <param name="max_vib" value="100" />
+            <param name="min_size" value="50" />
```

Comment:
I think it would be worth putting these in a yaml file

Reply:
Made this change